regolith/helpers/l_projectahelper.py

```diff
@@ -22,29 +22,36 @@
 TARGET_COLL = "projecta"
 HELPER_TARGET = "l_projecta"
 
+ACTIVE_STATI = ["proposed", "started"]
+
 
 def subparser(subpi):
     subpi.add_argument("--all", action="store_true",
-                       help="Lists all projecta that have not ended"
+                       help="Lists all projecta in general"
+                       )
+    subpi.add_argument("-c", "--current", action="store_true",
+                       help="Lists all active projecta")
+    subpi.add_argument("-v", "--verbose", action="store_true",
+                       help='increase verbosity of output'
                        )
-    subpi.add_argument("-v", "--verbose", action="store_true", help='increase verbosity of output')
     subpi.add_argument("-l", "--lead",
                        help="Filter milestones for this project lead"
                        )
     subpi.add_argument("-p", "--person",
                        help="Filter milestones for this person whether lead or not"
                        )
     subpi.add_argument("-e", "--ended", action="store_true",
-                       help="Lists projects that have ended. Use the -d and -r flags to specify"
-                            "from one date and how many days"
+                       help="Lists projecta that have ended. Use the -d and -r flags to specify"
+                            " up to what date and how many days around that"
                        )
     subpi.add_argument("-d", "--date",
                        help="projecta with end_date within RANGE before this date will be listed."
-                            "Default is today"
+                            " Default is today"
+                            " Be wary some projecta do not have a specified end date"
                        )
     subpi.add_argument("-r", "--range",
                        help="date range back from DATE to search over in days. If no "
-                            "range is specified, search will be 7 days"
+                            "range is specified, search will be 0 days"
```

Comment:
Is this correct?  I thought the desired default range was "ended in the last 7 days", so that when we run this with -e it will show all the prums that ended in the previous week, which is what we want to do each group meeting.

Reply:
Ah ok, if this is for just inside the group then yep I'll edit that.

Reply:
thanks

Reply:
fixed it to 7 days and changed up some of the other wording too